dev/seed.py

```diff
@@ -67,29 +102,43 @@ def load_data(count: int, db_path: str):
     db_session = create_session(db_url)
 
     with db_session() as session:
-        for index in range(1, count + 1):
-            session.add(create_random_job(index))
+        job_def_ids = []
+
+        for index in range(1, job_defs_count + 1):
+            job_def = create_random_job_def(index)
+            session.add(job_def)
+            job_def_ids.append(job_def.job_definition_id)
+
+        for index in range(1, jobs_count + 1):
+            session.add(create_random_job(index, random.choice(job_def_ids)))
 
         session.commit()
 
-    click.echo(f"\nCreated {count} jobs in the scheduler database")
-    click.echo(f"present at {db_path}, copy the following command")
+    click.echo(
+        f"\nCreated {jobs_count} jobs and {job_defs_count} job definitions in the scheduler database"
+    )
+    click.echo(f"present at {db_path}. Copy the following command")
     click.echo(f"to start JupyterLab with this database.\n")
     click.echo(f"`jupyter lab --SchedulerApp.db_url={db_url}`\n")
 
 
 @click.command(
     help="Inserts random jobs in the scheduler database. Note, that this command will drop the tables and re-create."
 )
-@click.option("--count", default=25, help="No of jobs to create, default is 25")
+
+# set to unique numbers by default to help test pagination with partially empty pages.
+@click.option("--jobs-count", default=57, help="No of jobs to create, default is 57.")
+@click.option(
+    "--job-defs-count", default=27, help="No of job definitions to create, default is 27."
+)
 @click.option(
     "--db_path",
     type=click.Path(),
     default=get_db_path(),
     help="DB file path, default is scheduler db path",
 )
```

Comment:
Can you check if there are shortcut options (e.g., -j -jd -db) for inputs and add them?

Reply:
Doing this is easy just by adding an additional positional argument to `@click.option()`. One small change though, we will be using double-hyphen (`--j`, `--jd`, `--db`) to specify these CLI arguments require an additional CLI argument after them, to conform to existing conventions.

https://serverfault.com/questions/387935/whats-the-difference-betwen-the-single-dash-and-double-dash-flags-on-shell-comm